src/Foundatio.AzureServiceBus/Messaging/AzureServiceBusMessageBus.cs

```diff
@@ -73,44 +71,34 @@ public class AzureServiceBusMessageBus : MessageBusBase<AzureServiceBusMessageBu
 
                 _subscriptionClient.RegisterMessageHandler(OnMessageAsync, new MessageHandlerOptions(OnExceptionAsync) { MaxConcurrentCalls = maxConcurrentCalls, AutoComplete = autoComplete });
                 sw.Stop();
-                _logger.Trace("Ensure topic subscription exists took {0}ms.", sw.ElapsedMilliseconds);
+                if (_logger.IsEnabled(LogLevel.Trace)) _logger.LogTrace("Ensure topic subscription exists took {ElapsedMilliseconds}", sw.ElapsedMilliseconds);
```

Comment:
Looks good, I learned yesterday that if you do `"{Duration:g}", sw.Elapsed` you get really nice formatting for timespans :)

Reply:
will make these changes. Thank you